Sources/OpenAPIRuntime/Conversion/CurrencyExtensions.swift

```diff
@@ -144,6 +144,32 @@ extension Converter {
         return HTTPBody(data)
     }
 
+    /// Returns a value decoded from a XML body.
+    /// - Parameter body: The body containing the raw XML bytes.
+    /// - Returns: A decoded value.
+    /// - Throws: An error if decoding from the body fails.
+    /// - Throws: An error if no custom coder is present for XML coding.
+    func convertXMLToBodyCodable<T: Decodable>(_ body: HTTPBody) async throws -> T {
+        guard let coder = configuration.customCoder(for: "application/xml") else {
```

Comment:
Please add a static let for xml on the `OpenAPIMIMEType` type, and use that here, so that we don't need to copy the raw MIME type for XML 4 times in this file.